apps/api/src/modules/sets/sets.routes.ts

```diff
@@ -1,14 +1,17 @@
-import { Slug } from '@algomart/schemas'
+import { Language, Slug } from '@algomart/schemas'
 import { SetsService } from '@algomart/shared/services'
 import { FastifyReply, FastifyRequest } from 'fastify'
 
 export async function getSet(
-  request: FastifyRequest<{ Params: Slug }>,
+  request: FastifyRequest<{ Params: Slug; Querystring: Language }>,
   reply: FastifyReply
 ) {
   const setsService = request.getContainer().get<SetsService>(SetsService.name)
   // TODO: get language from request
```

Comment:
Can remove this TODO, too 😄

Reply:
Can I, instead, add a TODO to remove the TODO